### PR TITLE
Interop-qe: install OCP+CNV+ODF and run CNV smoke tests

### PR DESCRIPTION
**Pull request by dshchedr**

### CODE REVIEW

File: ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.12.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: cnv-odf-tests-aws-ipi-ocp412
+  interval: 960h
+  steps:
+    cluster_profile: aws-interop-qe
+    env:
+      BASE_DOMAIN: cnv-qe.rhood.us
```
Comment (rabin-io) [inline ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.12.yaml:32]: Please change the base domain to `release-ci.cnv-qe.rhood.us`

**Review by dbasunag — APPROVED** [review]

**Pull request merged**